src/main/java/com/example/demo/repository/ProductRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.example.demo.repository;
+
+import com.example.demo.model.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ProductRepository extends JpaRepository<Product, Long> {
```

Comment:
can be annotated with @Repository for better readability but doesn't have to be, just fyi